features/scheduled_task_executor.feature

```diff
@@ -0,0 +1,44 @@
+Feature: Verifies that unhandled exceptions in scheduled tasks are reported to Bugsnag regardless of Scheduler configuration
+
+Scenario: By default a single-threaded ThreadPoolTaskScheduler is used
+    When I run spring boot "ScheduledTaskScenario" with the defaults
```

Comment:
We should also run all these tests using the plain spring fixture. This will test the new functionality with:
- A non-boot version of spring
- An older version of spring (4.2)